### ISSUE TITLE
Each alternate lang page must be added as separate <url>

### ISSUE THREAD

**Issue by codefuhrer** [RoumenDamianoff/laravel-sitemap#107]
See https://support.google.com/webmasters/answer/2620865?hl=en

> You must create a separate url element for each URL. Each url element must include a loc tag indicating the page URLs, and an xhtml:link rel="alternate" hreflang="XX" subelement for every alternate version of the page, including itself.

**Comment by RoumenDamianoff** [comment]
Yes, this is supported for a long time.

**Comment by codefuhrer** [comment]
Could you please point me at the docs or examples?
As far as I know, <url> is only added once, with attached alternate links.

**Comment by RoumenDamianoff** [comment]
Look at [this example](https://github.com/RoumenDamianoff/laravel-sitemap/wiki/Dynamic-sitemap), the ``$translations`` array has 3 alternate versions of the example page.

**Comment by codefuhrer** [comment]
Yes, as I said – it yields one `url` with one `loc` and three `xhtml:link`s. 
But, as said in [Google docs](https://support.google.com/webmasters/answer/2620865?hl=en), each localized version should also has its own `url` and `links`.

So, instead of one entry it should be three – for each of languages.

**Comment by RoumenDamianoff** [comment]
You need to do this in your own apps logic, if you save the relations between your pages and their alternates you just loop all of them and in the end you'll have exactly this.. but this package can't do "the magic" by itself by guessing this relations. 

I don't know if it's a good idea to add automatically all translations as new records, but I'm open for suggestions about this.

**Comment by codefuhrer** [comment]
There's no magic needed: all required URLs are already stored in `$translations`:

```php
$translations = [
    ['language' => 'fr', 'url' => URL::to('pageFr')],
    ['language' => 'de', 'url' => URL::to('pageDe')],
    ['language' => 'en', 'url' => URL::to('pageEn')],
];
foreach ($translations as $translation) {
     $sitemap->add($translation['url'], 'date', '0.9', 'monthly', [], null, $translations);
}
```
By the way, the current language must be in `<xhtml:link>`s too.

> I'm open for suggestions about this.

Maybe it should be a separate method?
```php
$sitemap->addLocalized(array $translations, $date, $priority, ... );
```
